CHANGELOG.yaml

```diff
@@ -1,3 +1,7 @@
+master:
+    fixed bugs:
+     - GH-2509 Fix do not store memory heavy objects in summary - instead give options
```

Comment:
This should be an english sentence. 😄

Reply:
Nope, still seems a little off. Can you read the last few changelogs in the file for a reference?